packages/hint-compat-api/src/helpers/compat-api.ts

```diff
@@ -0,0 +1,223 @@
+                    worstBrowserFeatureSupported.version_added = info.version_added;
+                }
+
+                if (!worstBrowserFeatureSupported.version_removed && info.version_removed === true) {
+                    worstBrowserFeatureSupported.version_removed = true;
+                }
+
+                if (!worstBrowserFeatureSupported.version_removed || worstBrowserFeatureSupported.version_removed && info.version_removed && info.version_removed < worstBrowserFeatureSupported.version_removed) {
+                    worstBrowserFeatureSupported.version_removed = info.version_removed;
+                }
+            });
+
+            return worstBrowserFeatureSupported;
+        }
+
+        return browserFeatureSupported as SimpleSupportStatement;
+    }
+    /* eslint-enable camelcase */
+
+    private isFeatureRequiredToTest(typedFeatureValue: CompatStatement & MDNTreeFilteredByBrowsers, browsers: BrowserSupportCollection): boolean {
+        return Object.entries(browsers).some(([browser, browserVersionsList]): boolean => {
+            if (!typedFeatureValue.__compat || !typedFeatureValue.__compat.support) {
+                return false;
+            }
+
+            const browserFeatureSupported = this.getWorstCaseSupportStatementFromInfo((typedFeatureValue.__compat.support as any)[browser]);
+
+            // If we don't have information about the compatibility, ignore.
+            if (!browserFeatureSupported) {
+                return false;
+            }
+
+            const { version_added: addedVersion, version_removed: removedVersion } = browserFeatureSupported;
+
+            if (this.isCheckingNotBroadlySupported) {
+                // Boolean check
+                if (typeof addedVersion === 'boolean' && addedVersion === false) {
+                    return true;
+                }
+
+                // Version check
+                if (typeof addedVersion !== 'boolean' && addedVersion && browserVersionsList[0] <= browserVersions.normalize(addedVersion)) {
+                    return true;
+                }
+            } else {
+                // Boolean check
+                if (typeof removedVersion === 'boolean' && removedVersion === true) {
+                    return true;
+                }
+
+                // Version check
+                if (typeof removedVersion !== 'boolean' && removedVersion && browserVersionsList[browserVersionsList.length - 1] >= browserVersions.normalize(removedVersion)) {
+                    return true;
+                }
+            }
+
+            return false;
+        });
+    }
+
+    public isBrowserToSupportPartOfBrowsersCollection(browsersToSupport: BrowserSupportCollection, browserToSupportName: string): boolean {
+        if (!Object.keys(browsersToSupport).some((browser) => {
```

Comment:
Move the condition to another method, it is a little bit difficult to read.

Reply:
I hope is better now:
```
const isBrowserInsideCollection = Object.keys(browsersToSupport).some((browser) => {
            return browser === browserToSupportName;
        })

        return isBrowserInsideCollection;
```